docs/troubleshooting-tips.md

```diff
@@ -8,6 +8,20 @@ import TabsConstants from '@site/core/TabsConstants';
 
 This page provides tips to troubleshoot issues you may have with Rancher Desktop.
 
+#### Q: How can I fix the Docker error when starting a container using the VS Code dev-containers extension with version >`v0.266`?
+
+**A:** There is a current workaround for users experiencing a Docker error when starting a container using the dev-containers extension for VS code versioned `v0.266` or later and Rancher Desktop `v1.8.1`. Disabling Wayland in the user settings will allow the container to spin up successfully. This can be accomplished by either unchecking the box in the `Settings` > `Extensions` > `Dev Containers` tab labelled `Dev > Containers: Mount Wayland Socket (Applies to All Profiles)`, or by adding a customized setting to your VS code user `settings.json`. An example of the disabled setting is provided below:
+
+```
+"customizations": {
```

Comment:
actually I found out that the setting doesn't really work in the config-file. it only works as a user setting and so can't be specified in the settings.json.